ledger-core/v2/virtual/integration/constructor_api_test.go

```diff
@@ -18,9 +18,8 @@ import (
 
 func Test_API_Create(t *testing.T) {
 	t.Log("C4837")
-	ctx := inslogger.TestContext(t)
-
 	server := utils.NewServer(t)
+	ctx := inslogger.TestContext(t)
```

Comment:
I was actually thinking of moving this before NewServer and adding ctx to NewServer

Reply:
then
1. please don't access logger before  calling NewServer
2. make it same for all other tests (this was a reason for change - to unify across all tests)